docs/api.rst

```diff
@@ -0,0 +1,17 @@
+Reactive API Documentation
+--------------------------
+
+
+.. toctree::
+
+    charms.reactive.decorators
+    charms.reactive.helpers
+    charms.reactive.relations
+    charms.reactive.bus
+    charms.reactive.cli
```

Comment:
Though, we also import them into the top-level `charms.reactive` module; maybe we should just commit to having that top-level module represent the public API and document from there.